src/lib/viewers/image/__tests__/ImageBaseViewer-test.js

```diff
@@ -648,4 +649,74 @@ describe('lib/viewers/image/ImageBaseViewer', () => {
             expect(imageBase.updateCursor).to.be.called;
         });
     });
+
+    describe('print()', () => {
+        beforeEach(() => {
+            stubs.execCommand = sandbox.stub();
+            stubs.focus = sandbox.stub();
+            stubs.print = sandbox.stub();
+            stubs.mockIframe = {
+                addEventListener() {},
+                contentWindow: {
+                    document: {
+                        execCommand: stubs.execCommand,
+                    },
+                    focus: stubs.focus,
+                    print: stubs.print,
+                },
+                contentDocument: {
+                    querySelectorAll: sandbox.stub().returns(containerEl.querySelectorAll('img')),
+                },
+                removeEventListener() {},
+            };
+
+            stubs.openContentInsideIframe = sandbox.stub(util, 'openContentInsideIframe').returns(stubs.mockIframe);
+            stubs.getName = sandbox.stub(Browser, 'getName');
+        });
+
+        it('should open the content inside an iframe, center, and focus', () => {
+            imageBase.print();
+            expect(stubs.openContentInsideIframe).to.be.called;
+            expect(imageBase.printImages[0].style.display).to.equal('block');
```

Comment:
Can we match the `style` attribute to the string we expect rather than each sub-attribute?